File: ABISS.py
import os
import subprocess

def clear_screen():
    # Clear screen for different OS
    os.system('cls' if os.name == 'nt' else 'clear')

def display_banner():
    print("* Make Sure That You Ran ABISS As Admin *")
    print("\n")
    print("    ▄████████ ▀█████████▄   ▄█     ▄████████    ▄████████ ")
    print("   ███    ███   ███    ███ ███    ███    ███   ███    ███ ")
    print("   ███    ███   ███    ███ ███▌   ███    █▀    ███    █▀  ")
    print("   ███    ███  ▄███▄▄▄██▀  ███▌   ███          ███        ")
    print(" ▀███████████ ▀▀███▀▀▀██▄  ███▌ ▀███████████ ▀███████████ ")
    print("   ███    ███   ███    ██▄ ███           ███          ███ ")
    print("   ███    ███   ███    ███ ███     ▄█    ███    ▄█    ███ ")
    print("   ███    █▀  ▄█████████▀  █▀    ▄████████▀   ▄████████▀  ")
    print("\n")

def display_menu():
    print("             ╦")
    print("             ║")
    print("             ╠═ Add A User")
    print("             ║")
    print("             ╠═ List Users")
    print("             ║")
    print("             ╠═ Change A Users Password")
    print("             ║")
    print("             ╠═ Exit")
    print("             ╩")
    print("\n")

def list_users():
    subprocess.run("net user", shell=True)

def add_user():
    username = input("Username: ")
    password = input("Password: ")
    subprocess.run(f'net user {username} {password} /Add', shell=True)
    print(f"Successfully created user: {username} with Password: {password}")

def change_user_password():
    username = input("Username: ")
    new_password = input("New Password: ")
    subprocess.run(f'net user {username} {new_password}', shell=True)
    print(f"Successfully changed {username}'s password to {new_password}")

def main():
    clear_screen()
    display_banner()
    while True:
        display_menu()
        option = input("Select an option: ")
        if option == "1":
            add_user()
        elif option == "2":
            list_users()
        elif option == "3":
            change_user_password()
        elif option == "4":
            break
        else:
            print("Invalid option. Please try again.")
        input("\nPress Enter to continue...")  # Pause before next menu

if __name__ == "__main__":
    main()

File: ABISS.js
const { execSync } = require('child_process');
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

// Function to clear the console screen
function clearScreen() {
    process.stdout.write('\x1Bc');
}

// Function to display the banner
function displayBanner() {
    console.log("* Make Sure That You Ran ABISS As Admin *\n");
    console.log("    ▄████████ ▀█████████▄   ▄█     ▄████████    ▄████████ ");
    console.log("   ███    ███   ███    ███ ███    ███    ███   ███    ███ ");
    console.log("   ███    ███   ███    ███ ███▌   ███    █▀    ███    █▀  ");
    console.log("   ███    ███  ▄███▄▄▄██▀  ███▌   ███          ███        ");
    console.log(" ▀███████████ ▀▀███▀▀▀██▄  ███▌ ▀███████████ ▀███████████ ");
    console.log("   ███    ███   ███    ██▄ ███           ███          ███ ");
    console.log("   ███    ███   ███    ███ ███     ▄█    ███    ▄█    ███ ");
    console.log("   ███    █▀  ▄█████████▀  █▀    ▄████████▀   ▄████████▀  \n");
}

// Function to display the menu
function displayMenu() {
    console.log("             ╦");
    console.log("             ║");
    console.log("             ╠═ Add A User");
    console.log("             ║");
    console.log("             ╠═ List Users");
    console.log("             ║");
    console.log("             ╠═ Change A Users Password");
    console.log("             ║");
    console.log("             ╠═ Exit");
    console.log("             ╩\n");
}

// Function to list users
function listUsers() {
    try {
        const output = execSync('net user').toString();
        console.log(output);
    } catch (error) {
        console.log("Error: Could not list users.");
    }
}

// Function to add a user
function addUser() {
    rl.question('Username: ', (username) => {
        rl.question('Password: ', (password) => {
            try {
                execSync(`net user ${username} ${password} /Add`);
                console.log(`Successfully created user: ${username} with Password: ${password}`);
                mainMenu();
            } catch (error) {
                console.log('Error: Could not add user.');
                mainMenu();
            }
        });
    });
}

// Function to change user password
function changeUserPassword() {
    rl.question('Username: ', (username) => {
        rl.question('New Password: ', (newPassword) => {
            try {
                execSync(`net user ${username} ${newPassword}`);
                console.log(`Successfully changed ${username}'s password to ${newPassword}`);
                mainMenu();
            } catch (error) {
                console.log('Error: Could not change password.');
                mainMenu();
            }
        });
    });
}

// Main function to handle menu options
function mainMenu() {
    displayBanner();
    displayMenu();
    rl.question('Select an option: ', (option) => {
        switch (option) {
            case '1':
                addUser();
                break;
            case '2':
                listUsers();
                break;
            case '3':
                changeUserPassword();
                break;
            case '4':
                console.log('Exiting...');
                rl.close();
                break;
            default:
                console.log('Invalid option. Please try again.');
                mainMenu();
        }
    });
}

// Start the program
clearScreen();
mainMenu();
